src/multimodel/model.jl

```diff
@@ -858,6 +858,13 @@ function check_convergence(storage, model::MultiModel, cfg; tol = nothing, extra
         err = max(e, err)
         offset += number_of_degrees_of_freedom(m) ÷ model_block_size(m)
     end
+
+    if converged
+        if haskey(tol_cfg, :global_check) && !isnothing(tol_cfg[:global_check])
```

Comment:
Maybe it could be called `global_convergence_function`? We could also pass on the `kwarg` that contain the time and so on since that is useful for some convergence checks.